src/providers/data_providers/googlePlacesBusinessImageProvider.ts

```diff
@@ -0,0 +1,25 @@
+import { BusinessInfoInput, Geocode } from '../../models/data_models/types';
+import { BaseBusinessImageProvider } from './baseBusinessImageProvider';
+import { GooglePlacesApiUtils } from '../../utils/googlePlacesApiUtils';
+import { Image, ImageProviderName } from '../../models/data_models/image';
+
+export class GooglePlacesBusinessImageProvider extends BaseBusinessImageProvider {
+	public async getBusinessImages(businessInfo: BusinessInfoInput): Promise<Array<Image>> {
+		const googlePlacesApi = await new GooglePlacesApiUtils(this.config).init();
+		return googlePlacesApi
+			.getPlaceID(businessInfo.address, businessInfo.name)
+			.then((id) => googlePlacesApi.getPlaceDetails(id))
+			.then((photoRefs) =>
+				Promise.all(photoRefs.map((photoRef) => googlePlacesApi.getPhoto(photoRef)).filter((x) => x))
+			)
+			.then((photoUrls) =>
+				photoUrls.map(
+					(url: any) =>
+						<Image>{
```

Comment:
The `<Image>` typecast shouldn't be necessary here (I expect that it was required due to the `: any` annotation for `url`, but once that is removed, this shouldn't be necessary, because the function is expected to return `Promise<Array<Image>>`

Reply:
![image](https://user-images.githubusercontent.com/23726221/115075780-9b891700-9ec9-11eb-891b-b1e913108028.png)

Typescript doesn't like me anymore :(